shift

```diff
@@ -67,6 +67,12 @@ sub_ps() { #                      - Print service information
     sub_compose ps
 }
 
+sub_pull() { #                      - Update the codebase to HEAD of https://github.com/Shift2Bikes/shift-docs/tree/master
+    echo "pulling most recent commit from https://github.com/Shift2Bikes/shift-docs/tree/master"
```

Comment:
Nothing connects the "git pull" to the url specified here. Either update the command to pull from that url or update the description to mention that it pulls from the current branch